lib/config/themes.dart

```diff
@@ -199,12 +199,6 @@ abstract class TwakeThemes {
           onSecondaryContainer: brightness == Brightness.light
               ? LinagoraSysColors.material().onSecondaryContainer
               : LinagoraSysColors.material().onSecondaryContainerDark,
-          background: brightness == Brightness.light
-              ? LinagoraSysColors.material().background
-              : LinagoraSysColors.material().backgroundDark,
-          onBackground: brightness == Brightness.light
-              ? LinagoraSysColors.material().onBackground
-              : LinagoraSysColors.material().onBackgroundDark,
```

Comment:
can you list change of these here, and explain more. We also need to update system design